packages/blocks/src/database-block/common/columns/rich-text/cell-renderer.ts

```diff
@@ -202,76 +184,41 @@ export class RichTextCellEditing extends BaseCellRenderer<Y.Text> {
     }
   `;
 
-  vEditor: AffineVEditor | null = null;
-
-  @query('.affine-database-rich-text')
-  private _container!: HTMLDivElement;
-
-  private init() {
-    const vEditor = this._onInitVEditor();
-    this.vEditor = vEditor;
-    this.column.captureSync();
-    this.disposables.add({
-      dispose: () => {
-        vEditor.unmount();
-      },
-    });
-  }
+  @query('rich-text')
+  private _richTextElement?: RichText;
 
-  protected override firstUpdated() {
-    this.init();
+  get vEditor() {
+    assertExists(this._richTextElement);
+    const vEditor = this._richTextElement.vEditor;
+    assertExists(vEditor);
+    return vEditor;
   }
 
   override connectedCallback() {
     super.connectedCallback();
-    if (this._container) {
-      this.init();
+    if (!this.value || typeof this.value === 'string') {
+      this._initYText(this.value);
     }
   }
 
+  override updated() {
```

Comment:
add event listener in updated? This may result in listening for keydown event more than once